.github/ISSUE_TEMPLATE/roadmap-template.yml

```diff
@@ -0,0 +1,29 @@
+name: Roadmap feature Request
+description: Suggest a roadmap feature for this project.
```

Comment:
```suggestion
Description: Proposing a new feature for the Azure Container Registry roadmap.```

Reply:
The template is more for the PMs on the service. Also, this is not only for the ACR roadmap but for everything CNSR manages (ACR, MAR, OSS integration into the services, etc.)